src/AppBundle/Resources/views/Ndr/Ndr/_ndr_row.html.twig

```diff
@@ -4,13 +4,13 @@
 
 <h2 class="heading-large">New deputy report</h2>
 
-<div class="behat-region-ndr-card card push-double--bottom">
+<div class="behat-region-ndr-card opg-card push-double--bottom">
     <span class="status status-corner {{ ndrStatus.status }}">{{ (ndrStatus.status) | trans({}, 'common') }}</span>
     <h3 class="heading-medium flush--top">{{ client.firstname }} {{ client.lastname }}</h3>
-    <div class="card-body soft-half--top">
+    <div class="soft-half--top">
```

Comment:
Should use `govuk-!-padding-top-3` instead of `soft-half--top` (in a few related places too)

Not a blocker though, since updating the HTML isn't strictly part of this ticket